Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.13
+
+RUN apt update && apt install -y libgdal-dev gdal-bin libpq5
+
+RUN apt info libgdal-dev
+
+ARG REQUIREMENTS_FILE=requirements.txt
+
+COPY requirements.txt requirements.txt
+
+# https://docs.astral.sh/uv/guides/integration/docker/#installing-uv
+COPY --from=ghcr.io/astral-sh/uv:0.6.16 /uv /uvx /bin/
```

Comment:
If we are going to use uv, I'd recommend migration `requirements.txt` to `pyproject.toml`first.

Also, let's use the latest version.

Reply:
I'm just adding Docker support, not changing to pyproject.toml :)

But good point on upgrading it, man things go so fast... I wasn't even copying this from an old template :zany_face: